website/docusaurus.config.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
           label: 'Docs',
           position: 'left',
           items: [
-            { label: '0.15.x', to: 'docs/introduction' },
+            { label: '0.16.x', to: 'docs/introduction' },
+            { label: '0.15.x', to: 'docs/0.16.x/introduction' },
```

Comment:
```suggestion
            { label: '0.15.x', to: 'docs/0.15.x/introduction' },
```

Reply:
Added a minor suggestion. 

Can we also update the version in the release note? From `0.16` to `0.16.0`?

Reply:
Added that in https://github.com/aerogear/graphback/pull/1971/commits/56266d28e301367f49788d17b5a86a592ae107f5

Reply:
(Decided it would be good to have it grouped accordingly for merge)